presto-main/src/main/java/io/prestosql/server/PluginClassLoader.java

```diff
@@ -51,7 +51,7 @@ private PluginClassLoader(
             Iterable<String> spiResources)
```

Comment:
Can you add an explanation to the commit message? This is lacking context.

Reply:
Ping.